modules/mobile/app/controllers/mobile/v0/appointment_request_messages_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_dependency 'mobile/application_controller'
+
+module Mobile
+  module V0
+    class AppointmentRequestMessagesController < ApplicationController
```

Comment:
I named it this because we already have a messages controller that does something else.